lambda_src/drivers/process_https.py

```diff
@@ -98,10 +98,23 @@ def process_row(
     req_method: str = method.upper()
 
     if json:
+        
         req_data: Optional[bytes] = (
             json if json.startswith('{') else dumps(parse_header_dict(json))
         ).encode()
         req_headers['Content-Type'] = 'application/json'
+        if json.startswith('arn'):
```

Comment:
this check already exists in `decrypt_if_encrypted` -- we want it there since there's no need for this code to worry about the details of how secrets are stored.